base/expr.jl

```diff
@@ -118,6 +118,20 @@ macro eval(x)
     :($(esc(:eval))($(Expr(:quote,x))))
 end
 
+"""
+    @inline
+
+Small functions typically don't even need the @inline annotation,
+it does it on its own. By using @inline on bigger functions,
+an extra nudge can be given to the compiler, like in the example below.
+```
+    @inline function bigfunction(x)
+        #=
+           Function Definition
+        =#
+    end
```

Comment:
The content/meaning of a docstring and it's presentation to users should remain separate from each other. Adding an indent is just part of the presentation, and doesn't add any real meaning to the docstring in this case and so should not be included here.

If you'd like to propose global changes to the display of markdown objects in the REPL then that's fine (I think a small indent for code blocks would be nice perhaps), but that should be a separate PR where you make the necessary changes to the rendering code in https://github.com/JuliaLang/julia/tree/7ba6ad64d261307ce02fdca2edbe8aeca16ec1be/base/markdown/render/terminal so that all code blocks are presented to users in a uniform way rather than just this one.